### PR TITLE
Photo upload

### PR DESCRIPTION
**Pull request by DuncanRose**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -52,6 +52,10 @@ gem 'launchy', group: [:development, :test]
 gem 'factory_girl', group: [:development, :test]
 gem 'valid_attribute', group: [:development, :test]
 gem 'email_validator', '~> 1.5.0', group: [:development, :test]
+gem "carrierwave", group: [:development, :test]
+gem 'simple_form', group: [:development, :test]
+gem 'fog', group: [:development, :test]
+gem "dotenv-rails", group: [:development, :test]
 gem 'shoulda-matchers', group: [:development, :test], require: false
```
Comment (radavis) [inline Gemfile]: Would you guys mind cleaning up the Gemfile? Move everything that has `, group: [:development, :test]` to the block.

File: app/uploaders/profile_photo_uploader.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  # storage :file
+  # storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
```
Comment (radavis) [inline app/uploaders/profile_photo_uploader.rb]: Clean up this file by deleting comments.

File: config/initializers/carrierwave.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: "AWS",
+    aws_access_key_id: ENV["AWS_ACCESS_KEY_ID"],
+    aws_secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"]
+  }
+  config.fog_directory = ENV["S3_BUCKET"]
+end
```
Comment (radavis) [inline config/initializers/carrierwave.rb]: Travis is going to give you errors on these ENV variables. Create a `.env.example` file and copy it over to `.env` from your `.travis.yml`. Or, set them equal to something other than `nil`:

```
aws_access_key_id: ENV["AWS_ACCESS_KEY_ID"] || "",
```

**Comment by radavis** [comment]
Travis build is failing due to unassigned ENV variables.

Also, where's the `_spec.rb'?

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    if @user.save
+      flash[:notice] = 'Admin Successfully Created'
+
+      redirect_to users_path
+    else
+      flash[:errors] = @user.errors.full_messages
+
+      render
+    end
+  end
```
Comment (cmkoller) [inline app/controllers/users_controller.rb]: There are a BUNCH of new actions here that I don't see *any* tests for! If you're adding in all this new user functionality, we have to have tests to go along with it.

**Comment by cmkoller** [comment]
This branch is on photo upload, but I don't see any tests on uploading photos! This needs to be fixed.

File: Gemfile (hunk 1 above)
Comment (cocolote) [inline Gemfile]: The Gemfile was cleaned in a different branch

File: app/controllers/users_controller.rb (hunk 4 above)
Comment (cocolote) [inline app/controllers/users_controller.rb]: HI Christina. It seems that we have the tests in the add_admin branch but did the revisions to functionality in this branch by mistake. https://github.com/DuncanRose/MovieHub/blob/master/spec/features/admins/add_new_admin_spec.rb... oops

**Pull request merged**